context/app/static/js/components/workspaces/WorkspaceToastSuccessMessage.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+
+function WorkspaceToastSuccessMessage(id: number) {
+  return (
+    <Stack sx={{ maxWidth: '22rem' }}>
+      <Typography>
+        Workspace successfully launched in a new tab. If the tab didn&apos;t open, please check your pop-up blocker
+        settings and relaunch your workspace.
+      </Typography>
+      <Stack direction="row" justifyContent="flex-end" marginTop={1}>
+        <Button
+          href={`/workspaces/${id}`}
+          variant="contained"
```

Comment:
Would `variant="text"` be more appropriate? We likely wouldn't need many of the below styles.

Reply:
Yes, that works - the weird styling was the result of attempting to change the button text color, which it seems can only be changed to a small number of themed options. I switched to using variant="text" and color="inherit", which worked.